app/controllers/users_controller.rb

```diff
@@ -48,4 +49,8 @@ def photo_galleries
     @galleries = @user.photo_galleries
   end
 
+  def latest_photo_comments
+    @latest_photo_comments = @user.latest_photo_comments.limit(10)
+  end
+
```

Comment:
Also I fail to see why this would be a controller method. Can you not just use "@user.late..(10)" in the views? Or else, would it make sense to write a helper to be used in the views?